extensions/Worker.Extensions.Http.AspNetCore/src/FunctionsMiddleware/FunctionsHttpProxyingMiddleware.cs

```diff
@@ -43,7 +43,7 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
 
         private static void AddHttpContextToFunctionContext(FunctionContext funcContext, HttpContext httpContext)
         {
-            funcContext.Items.Add("HttpRequestContext", httpContext);
+            funcContext.Items.Add(Constants.HttpContextKey, httpContext);
```

Comment:
Consider using `IInvocationFeatures` instead. (`FunctionContext.Features`) - as I see `HttpContext` being a feature of this invocation.

Reply:
Agreed. Doesn't have to be part of this PR, so we can track this separately.